File: Modulo09/Triangulo-CSharp/Equilatero.cs
using System;

namespace poo {

    public class Equilatero : Triangulo {

        public Equilatero(double lado) : base(lado, lado, lado) {
        }

        public override double getArea() {
            return lado1 * lado1 * Math.Sqrt(3) / 4;
        }
    }

}

File: Modulo09/Triangulo-v2/Equilatero.cpp
#include "Equilatero.h"

Equilatero::Equilatero(double lado) :
Triangulo(lado, lado, lado) {
}

double Equilatero::getArea() const {
    cout << "Invocando Equilatero" << endl;
    return lado1 * lado1 * sqrt(3) / 4;
}


